tests/Countries/EcuadorTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Spatie\Holidays\Tests\Countries;
+
+use Carbon\CarbonImmutable;
+use Spatie\Holidays\Holidays;
+
+it('can calculate ecuador holidays', function () {
```

Comment:
Can you also add a test to cover the translation?

Reply:
Sure, but I had to change `defaultLocale()` to `en` to test Spanish and English translations, If I set the `defaultLocale()` to `es` I cannot test the English translations

Reply:
By running `./vendor/bin/pest --update-snapshots` you can update the snapshots. Just make sure the generated results are correct.

Reply:
Sure I just did it, the snapshots are correct now.